pkg/apis/kappctrl/v1alpha1/types_template.go

```diff
@@ -21,6 +22,21 @@ type AppTemplateYtt struct {
 
 type AppTemplateKbld struct{}
 
+type AppTemplateSopsArgs struct {
+	KMSKeys    []string `json:"kmsKeys,omitempty"`
+	GCPKms     []string `json:"gcpKms,omitempty"`
+	AzureKV    []string `json:"azureKV,omitempty"`
+	PGP        []string `json:"pgp,omitempty"`
+	AWSProifle string   `json:"awsProfile,omitempty"`
+	IgnoreMac  bool     `json:"ignoreMac,omitempty"`
+}
+
+type AppTemplateSops struct {
+	Match      string               `json:"match,omitempty"`
+	MergeFiles bool                 `json:"mergeFiles,omitempty"`
```

Comment:
do we need to have Match and MergeFiles functionality? can we simplify this integration to having sops just rewrite files on disk that are encrypted by it? i m guessing sops "templater" will always be used as a first templater so we probably shouldnt worry about merging of files.